CONTRIBUTING_DOCS.md

```diff
@@ -132,27 +132,50 @@ Examples:
 - Calico API schema reference (policy, ip pool, etcd)
 
 
-## Doc site architecture
+## Organizational changes
 
-The naming and layout of these navbars are stored in `_data/$VERSION/navbars/*`. Jekyll automatically stores information from the `_data` directory in an accessible variable called `site.data`. The top-level layout (`_layout/docwithnav.html`) will iterate through all the files in `site.data[version].navbars` to construct the sidebar based on which version is being viewed.
+### Creating new pages
+
+- To create a top level splash page for a URL path, simply name the file `index.md`.
```

Comment:
While you're here, can you point out that they'll "need to add a sidebar entry as well for new pages (see below)"